### DISCUSSION TITLE
On the header block should we support 'image origin' options

### DISCUSSION THREAD

**Discussion by SophieAmnesty** [amnestywebsite/humanity-theme#79] [Enhancement Requests]
We sometimes struggle with the positioning of images in the home page banner. Either because of the way an image is cropped differently when being viewed on different screen sizes, or because of the way the text overlays the image and obscures certain parts. 

e.g.: 
![image](https://user-images.githubusercontent.com/99269799/201311731-9b7c1b89-2c20-47ea-9019-0006a8d2a2c5.png)

Obviously a part of this is image choice. 

On the section block we have the option to select the 'image origin' and position the image in the container. 

![MicrosoftTeams-image (31)](https://user-images.githubusercontent.com/99269799/201312857-a216d91f-bd4f-4103-8645-ad0fea4a8ba6.png)

Just wondered whether this would be a useful feature to add to the Header block as well. 

I realize this may not be a solution we go with but just popping this in a ticket so that we think about it the next time we spend time on the Home Page.

**Comment by AmnestyAM** [comment]
Good idea, for now you could just crop a chunk off the left or right of the image to force the main part of the image to the side. Thanks
